app/notifications/pair_request/completion_reminder_notification.rb

```diff
@@ -0,0 +1,20 @@
+class PairRequest::CompletionReminderNotification < Noticed::Base
+  deliver_by :email,
+    mailer: 'PairRequestMailer',
+    method: :remind_for_completion,
+    delay: :now_to_end_of_meeting,
+    if: :pair_request_accepted?
+
+  param :pair_request
+
+  private
+
+  def now_to_end_of_meeting
+    end_time = params[:pair_request].end_time
+    ActiveSupport::Duration.build(end_time - Time.current)
+  end
+
+  def pair_request_accepted?
+    params[:pair_request].accepted?
```

Comment:
Intentionally checking for the pair request being in an `accepted` state here. If I were to check for "not complete", it could open up edge case issues with pair requests that have been canceled or rejected.